src/pages/Storefront/Cart.js

```diff
@@ -0,0 +1,119 @@
+import React, { useEffect } from 'react';
+import { useShopify } from '../../hooks/useShopify';
+import { CartItem } from 'components/storefront';
+import { Box, HStack, VStack, Heading, Divider, Flex, FormControl, Text, Button, Input } from '@chakra-ui/react';
+import { useSelector, shallowEqual } from 'react-redux';
+import { selectors as teamSelectors } from '../../data/reducers/team';
+
+// Cart page that shows the user the items in their cart, allows them to change quantity, and proceed to checkout
+
+const Cart = () => {
+  const { products, checkoutState, updateCartAttributes } = useShopify();
+  const team = useSelector(teamSelectors.selectTeam, shallowEqual);
+
+  /* previous code that might be userful
+  useEffect(() => {
+    const userID = sessionStorage.getItem('userID');
+    updateCartAttributes(checkoutState.id, [
+      { key: 'userID', value: userID ? userID.toString() : '1' }, // Temporary, so that page doesn't crash
+      // Remove once user info is added to redux
+      { key: 'teamID', value: team.id.toString() },
+      { key: 'teamName', value: team.name },
+    ]);
+  }, [checkoutState.id, team.id, team.name]); // eslint-disable-line react-hooks/exhaustive-deps
+  */
+
+  // Temporary checkout code from last term, will need to udpate
+  const openCheckout = () => {
+    if (checkoutState.webUrl) {
+      console.log('webUrl exists');
+    }
+    // window.open(checkoutState.webUrl) // opens checkout in a new window
+    window.location.replace(checkoutState.webUrl); // opens checkout in same window
+  };
+
+  const applyCoupon = () => {
+    alert('To be implemented');
+  };
+
+  return (
+    <>
```

Comment:
Some of the spacing might be slightly different than the Figma. I also wasn't too familiar with the mappings for the values for the spacings (e.g. I wasn't sure what x maps to in terms of px inside p={x}) so I used `px`, I can definitely fix this.

Reply:
I think our codebase uses both `px` and Chakra spacing values so not sure if it's necessary to change but the mappings can be found here: https://chakra-ui.com/docs/theming/theme#spacing

Reply:
yeah, it's a bit of a mess in our codebase just use whatever you'd like and whatever looks good to your eye